### PR TITLE
DOMA-4476 -- Multipayment support for partial payments

### PR DESCRIPTION
**Pull request by toplenboren**
Now we can support partial payments in RegisterMultiPayment mutation.

### CODE REVIEW

File: apps/condo/domains/billing/utils/serverSchema/index.js [hunk 1]
```diff
@@ -20,9 +20,11 @@ const { ResidentBillingReceipt: ResidentBillingReceiptGQL } = require('@condo/do
 const { BillingCurrency: BillingCurrencyGQL } = require('@condo/domains/billing/gql')
 const { BillingRecipient: BillingRecipientGQL } = require('@condo/domains/billing/gql')
 const { execGqlWithoutAccess } = require('@condo/codegen/generate.server.utils')
-const { Payment } = require('@condo/domains/acquiring/gql')
 const { BillingCategory: BillingCategoryGQL } = require('@condo/domains/billing/gql')
 const { REGISTER_BILLING_RECEIPTS_MUTATION } = require('@condo/domains/billing/gql')
+const { find } = require('@condo/keystone/schema')
+const { PAYMENT_DONE_STATUS, PAYMENT_WITHDRAWN_STATUS } = require('@condo/domains/acquiring/constants/payment')
+const Big = require('big.js')
 /* AUTOGENERATE MARKER <IMPORT> */
```
Comment (Kanol) [inline apps/condo/domains/billing/utils/serverSchema/index.js:28]: reorder imports

File: apps/condo/domains/billing/utils/serverSchema/index.js [hunk 2]
```diff
@@ -57,6 +59,28 @@ async function registerBillingReceipts (context, data) {
...
+ * @param context {Object}
+ * @param organizationId {string}
+ * @param accountNumber {string}
+ * @param bic {string}
+ * @param bankAccount {string}
+ * @param period {string}
+ * @return {Promise<*>}
+ */
+const getPaymentsSum = async (context, organizationId, accountNumber, period, bic, bankAccount) => {
+    const payments = await  find('Payment', {
```
Comment (Kanol) [inline apps/condo/domains/billing/utils/serverSchema/index.js:73]: Does it work on more than 100 payments?

Comment (toplenboren) [reply apps/condo/domains/billing/utils/serverSchema/index.js:73]: yep, find works on entities with more then 100 paymenst

Comment (sitozzz) [inline apps/condo/domains/billing/utils/serverSchema/index.js:73]:
```suggestion
    const payments = await find('Payment', {
```

File: apps/condo/domains/billing/utils/serverSchema/index.js [hunk 3]
```diff
@@ -57,6 +59,28 @@ async function registerBillingReceipts (context, data) {
     })
 }
 
+/**
+ * Sums all DONE or WITHDRAWN payments for billingReceipt for <organization> with <accountNumber> and <period>
```
Comment (sitozzz) [inline apps/condo/domains/billing/utils/serverSchema/index.js:63]:
```suggestion
 * Sums up all DONE or WITHDRAWN payments for billingReceipt for <organization> with <accountNumber> and <period>
```

**Review by sitozzz — APPROVED** [review]

**Review by Kanol — APPROVED** [review]

**Pull request merged**